WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -130,28 +130,24 @@ - (void)viewDidLoad
 
 - (void)setupHeaderView
```

Comment:
I realize it was already this way, but if its not too much trouble, maybe move `setupHeaderView` and `updateHeaderSize` below all the view controller's view management and rotation methods?  I don't think we have anything specific in the style guide about grouping the view controller / view life cycle methods, but the grouping does make them easier to find.